### PR TITLE
CLOUDP-68758: Include API keys in creation message

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-68758

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

When we create an API key this is the only time we get to see the private key, so we need to show it

### CODE REVIEW

File: internal/cli/projects/apikeys/assign.go [hunk 1]
```diff
@@ -11,57 +11,57 @@
...
-	fmt.Println("Successfully updated APIKey.")
-	return nil
+	return output.Print(config.Default(), updateTemplate, nil)
 }
 
-// mongocli iam project(s) apiKey(s)|apikey(s) update <ID> [--role role][--projectId projectId]
-func UpdateBuilder() *cobra.Command {
-	opts := new(UpdateOpts)
+// mongocli iam project(s) apiKey(s)|apikey(s) assign <ID> [--role role][--projectId projectId]
+func AssignBuilder() *cobra.Command {
```
Comment (gssbzn) [inline internal/cli/projects/apikeys/assign.go:59]: I saw this command is actually to assign the key to a project so make this clearer for users

File: internal/cli/projects/apikeys/assign.go [hunk 2]
```diff
@@ -11,57 +11,57 @@
...
+	opts := new(AssignOpts)
 	cmd := &cobra.Command{
-		Use:     "update <ID>",
-		Aliases: []string{"updates"},
-		Args:    cobra.ExactArgs(1),
-		Short:   description.ProjectOrganizationsAPIKey,
+		Use:   "assign <ID>",
```
Comment (colm-quinn) [inline internal/cli/projects/apikeys/assign.go:62]: [q] Do we need to keep an alias to the old update command here, so users of the current version?

Comment (gssbzn) [reply internal/cli/projects/apikeys/assign.go:62]: I thought about it but we have not released this so I was ok with just a full rename

Comment (colm-quinn) [reply internal/cli/projects/apikeys/assign.go:62]: sgtm, thanks for the clarification.

**Pull request merged**